documentation-site/pages/components/list.mdx

```diff
@@ -50,37 +55,6 @@ List items best used in mobile applications.
   <MenuAdapter />
 </Example>
 
-## Overrides
-
-<Overrides
```

Comment:
Overrides / API imports should be deleted

Reply:
overrides are deleted in this diff, and imports [are maintained](https://baseweb.design/components/button/#api) correct?

Reply:
You should delete 

```
import API from '../../components/api';
import Overrides from '../../components/overrides';
```

from this file since it's not used anymore?

Reply:
oh right